notebooks/EDA Climate.ipynb

```diff
@@ -0,0 +1,1821 @@
+{
```

Comment:
<p>If there's a sign change, the model a priori can't be multiplicative in the classical sense. No need to futz with numerical tricks with no principled foundation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jessegrabowski/laos-climate-change/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Okay that makes sense. The reason why I was doing this is just because the units are so wonky. The unit is "Ocean heat anomaly 10^21 Joules"